Add printing style sheet, enables printing of text files

susnux opened the pull request:
* Resolves: #112
* Target version: master 

### Summary
Currently printing will include the menu bar and longer text files will be cropped after the first page.
This enables printing of text documents without any major styling changes,
slightly adjusted margins and added table borders (they are quite hard to read without on paper).

This implements the CSS part of #112

vinicius73 commented on css/print.scss:
```diff
@@ -0,0 +1,83 @@
+		display: none;
+	}
+
+	#viewer {
```
We should avoid global selector like that.

```suggestion
	#viewer[data-handler="text"] {
```

susnux replied:
Fixed that

susnux commented:
I had to add the rule for hiding the content behind the modal as an event listener to prevent hiding the content if the editor was loaded for *rich workspace*.
So this should now work for all cases.

vinicius73 commented:
Looks good @susnux 

But only the "first page" is printable.

https://user-images.githubusercontent.com/1561347/189185433-fb556c0f-cee4-4fa7-a6e6-ae3a85ba5662.mp4

This is expected?

File used into this test:

[demo.md](https://github.com/nextcloud/text/files/9529021/demo.md)

susnux commented:
@vinicius73 
> But only the "first page" is printable.
> Is this expected?

No, this happened because the `body` was set to `position: fixed` and `overflow: hidden`. Added a rule to fix this.

vinicius73 reviewed (approved):
Looks awesome.

https://user-images.githubusercontent.com/1561347/189776390-9ffb7e1c-b276-4665-8460-da83ecfb60ea.mp4

vinicius73 commented:
/compile
